frontend/src/components/PairRoom/TimerCard/TimerEditPanel/TimerEditPanel.tsx

```diff
@@ -0,0 +1,75 @@
+import { useParams } from 'react-router-dom';
+
+import Button from '@/components/common/Button/Button';
+import Input from '@/components/common/Input/Input';
+
+import useToastStore from '@/stores/toastStore';
+
+import useInput from '@/hooks/common/useInput';
+import useModal from '@/hooks/common/useModal';
+
+import useAddTimer from '@/queries/PairRoomOnboarding/useAddTimer';
+
+import { validateTime } from '@/utils/PairRoomOnboarding/validate';
+
+import * as S from './TimerEditPanel.styles';
+
+interface TimerEditPanelProps {
+  isActive: boolean;
+}
+
+const TimerEditPanel = ({ isActive }: TimerEditPanelProps) => {
+  const { accessCode } = useParams();
+  const { addToast } = useToastStore();
+
+  const { isModalOpen: isPanelOpen, openModal: openPanel, closeModal: closePanel } = useModal();
+  const { value, handleChange, resetValue } = useInput();
+  const { handleAddTimer } = useAddTimer(() =>
+    addToast({ status: 'SUCCESS', message: '타이머 시간이 정상적으로 수정되었습니다.' }),
```

Comment:
사소한 거긴 한데 '정상적으로' 는 빠져도 될 것 같아요 🤗
```suggestion
    addToast({ status: 'SUCCESS', message: '타이머 시간이 수정되었습니다.' }),
```

Reply:
좋습니다 🚀